src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsallocations/service/GetUnallocatedCaseService.kt

```diff
@@ -174,16 +174,16 @@ class GetUnallocatedCaseService(
     Flux.fromIterable(unallocatedCasesRepository.getCaseCountByTeam(teamCodes))
       .map { CaseCountByTeam(it.getTeamCode(), it.getCaseCount()) }
 
-  fun getChooseOffenderManagerCase(crn: String, convictionId: Long): ChooseOffenderManagerCase? =
+  fun getChooseOffenderManagerCase(crn: String, convictionNumber: Long): ChooseOffenderManagerCase? =
     findUnallocatedCaseByConvictionIdOrConvictionNumber(
       crn,
-      convictionId
+      convictionNumber
     )?.let { from(it) }
 
   private fun findUnallocatedCaseByConvictionIdOrConvictionNumber(
     crn: String,
     convictionId: Long
-  ) = if (convictionId < 100) {
+  ) = if (convictionId < 10000) {
```

Comment:
Found a PoP with over 100 convictions the other day!
We should remove this really